### PR TITLE
corrected db of contacts

### PR DESCRIPTION
**Pull request by lewiskg**

### CODE REVIEW

File: db/contacts.json [hunk 1]
```diff
@@ -1,93 +1,102 @@
...
+					"firstName": "Penny",
+					"lastName": "Hofstadter",
+					"phoneHome": "555-555-5545",
+					"phoneWork": "555-555-3450",
+					"emailHome": "penny.hofstadter@gmail.com",
+					"emailWork": "penny.hofstadter@cheescakefactory.com",
+					"notes":	"Leondard is better than nothing.",
+					"uid": "Wjpi45FakkThV2JHN2SzauwXqDg1"
+				},
+	"contact4": {	"id": 4,
```
Comment (morecallan) [inline db/contacts.json]: You won't need these id's as the key (in this case 'contact4' will serve as the id later on).

**Review by morecallan — COMMENTED** [review]
@lewiskg - nice proactive work here! One small comment. Once you take care of that, we'll merge it in!

**Comment by rousell** [comment]
@lewiskg those changes look good! one more thing though, we don't want to have any dummy uids in the seed data. You can alter those record to use my uid as registered for your application: J5y1dQQRMPZoD9Pc7OKe8fTziUr2. then i swear we will merge this in.

**Comment by rousell** [comment]
@lewiskg 👍 merging in

**Pull request merged**